README.md

```diff
@@ -1,3 +1,39 @@
-# coSMicQC
+<img height="200" src="https://raw.githubusercontent.com/WayScience/coSMicQC/main/logo/with-text-for-light-bg.png?raw=true">
 
-coSMicQC is a software tool for single-cell quality control of morphology datasets.
+# Single cell Morphology Quality Control
+
+[![Build Status](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml/badge.svg?branch=main)](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml?query=branch%3Amain)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+
+> :stars: Navigate the cosmos of single-cell morphology with confidence — coSMicQC keeps your data on course!
+
+coSMicQC is a software tool to evaluate converted single-cell morphology outputs from CytoTable.
+
+When performing object detection in CellProfiler, it is very hard to get "perfect" compartment segmentation across a large high-throughput screen.
+Technical artifacts can occur during segmentation, including under or over segmentation or segmentation of background or smudges/bright artifacts.
+
+In single-cell analysis, interesting phenotypes often emerge by looking at the morphological differences. However, the presence of technical outliers in the data can obscure these biological insights, undermining the validity of the findings.
```

Comment:
```suggestion
In single-cell analysis, intriguing phenotypes often emerge by examining morphological differences.
However, technical outliers in the data can obscure these biological insights, compromising the validity of the findings.
```

Reply:
I like how this is more concise, updating now!